src/components/ClickableStyle/ClickableStyle.stories.tsx

```diff
@@ -0,0 +1,268 @@
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+
+import { ClickableStyle, ClickableStyleProps } from './ClickableStyle';
+import Icon from '../Icon';
+
+export default {
+  title: 'Molecules/Buttons/ClickableStyle',
+  component: ClickableStyle,
+} as Meta;
+
+const Template: Story<ClickableStyleProps<React.ElementType>> = (args) => (
+  <ClickableStyle {...args} as="button" />
+);
+
+export const Primary = Template.bind({});
```

Comment:
we got rid of the `ClickableStyle` stories in EDS `main`, do we want them back now?

Reply:
I was mostly using them for testing and didn't think too much about whether we should keep them. I actually do wonder if we should have them because it is possible for devs to use it in `traject` for `NavLink`s and such, but then we should probably add it to zeroheight too. What do you think?

Reply:
I'm going to go ahead and merge this for now, but let's talk about this some more later outside of this PR.